src/StackDemo.java

```diff
@@ -0,0 +1,41 @@
+public class StackDemo{
+	
+	public static void main(String args[]){
+		char data[] = {'a','b','c','d','e','f','g','h','i','j','k'};
+		Stack stack = new Stack(10);
+		for(char ch : data) stack.push(ch);
+		try{
+		  for (int i = 0; i < 11; i++) System.out.println("Character " + i + " = "+stack.pop());
+		}
+		catch (StackUnderflowException e){
+		  System.out.println("Stack Underflow value no character could be returned.");
+		}
+	}	
+}	
+class StackOverflowException extends Throwable {}
+class StackUnderflowException extends Throwable {}
+
+class Stack {
+	
+  private char data[];
+  private int position=0;
+  
+
+  
+  Stack (int size) { data = new char[size]; }
+  
+  void push (char ch)
```

Comment:
Would have preferred to see this method throw the exception rather than throw it, catch it and display an error message.